graspologic/plot/plot.py

```diff
@@ -1267,6 +1273,8 @@ def _unique_like(vals):
 # assume that the graph has already been plotted in sorted form
 def _plot_groups(ax, graph, inner_labels, outer_labels=None, fontsize=30):
     inner_labels = np.array(inner_labels)
+    if outer_labels is not None:
+        outer_labels = np.array(outer_labels)
```

Comment:
I saw some other code that appeared to do this, but in practice didn't, because an error was getting raised when I tried to throw in `outer_labels` as a list, which is why I added this in